### PR TITLE
edit staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutation/dispatcher.go to structured Log

### PR DESCRIPTION
**Pull request by LinshanYu**
/kind documentation

edit staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutation/dispatcher.go to structured Log，https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

### CODE REVIEW

**Comment by jpbetz** [comment]
/lgtm

The mapping of `Warningf` -> `InfoS` matches the [migration guidelines](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md)

**Comment by jpbetz** [comment]
This needs an approver. @sttts would you be willing to approve?

File: staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go [hunk 1]
```diff
@@ -167,7 +167,7 @@ func (a *mutatingDispatcher) Dispatch(ctx context.Context, attr admission.Attrib
 
 		if callErr, ok := err.(*webhookutil.ErrCallingWebhook); ok {
 			if ignoreClientCallFailures {
-				klog.Warningf("Failed calling webhook, failing open %v: %v", hook.Name, callErr)
+				klog.InfoS("Failed calling webhook, failing open %v: %v", hook.Name, callErr)
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go:170]: I think it is the wrong format of InfoS.

File: staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go [hunk 2]
```diff
@@ -179,7 +179,7 @@ func (a *mutatingDispatcher) Dispatch(ctx context.Context, attr admission.Attrib
 					continue
 				}
 			}
-			klog.Warningf("Failed calling webhook, failing closed %v: %v", hook.Name, err)
+			klog.InfoS("Failed calling webhook, failing closed %v: %v", hook.Name, err)
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go:182]: ditto

File: staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go [hunk 3]
```diff
@@ -265,7 +265,7 @@ func (a *mutatingDispatcher) callAttrMutatingHook(ctx context.Context, h *admiss
 	for k, v := range result.AuditAnnotations {
 		key := h.Name + "/" + k
 		if err := attr.Attributes.AddAnnotation(key, v); err != nil {
-			klog.Warningf("Failed to set admission audit annotation %s to %s for mutating webhook %s: %v", key, v, h.Name, err)
+			klog.InfoS("Failed to set admission audit annotation %s to %s for mutating webhook %s: %v", key, v, h.Name, err)
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go:268]: The rest are the same .

**Comment by yangjunmyfm192085** [comment]
/wg structured-logging
@LinshanYu, do you still work on this pr?

**Pull request closed without merging**